node_modules/col-assets/tests/test-assets.js

```diff
@@ -920,6 +921,58 @@ describe('Assets', function() {
     });
 
     /**
+     * Test that verifies that an asset can be deleted by a user under certain conditions
+     */
+    it('can be deleted by user', function(callback) {
+      TestsUtil.getAssetLibraryClient(null, null, null, function(client1, course, user1) {
+        AssetsTestUtil.assertCreateLink(client1, course, 'UC Berkeley', 'http://www.ucberkeley.edu/', null, function(asset1) {
+
+          // Verify that a different user cannot delete the asset
+          TestsUtil.getAssetLibraryClient(null, course, null, function(client2, course, user2) {
+            AssetsTestUtil.assertDeleteAssetFails(client2, course, asset1.id, 401, function() {
+              //Verify that the creator can delete the asset
+              AssetsTestUtil.assertDeleteAsset(client1, course, asset1.id, function() {
+
+                // Create an asset with comments
+                AssetsTestUtil.assertCreateLink(client1, course, 'UC Davis', 'http://www.ucdavis.edu/', null, function(asset2) {
+                  AssetsTestUtil.assertCreateComment(client2, course, asset2.id, 'Comment 1', null, function(comment) {
+                    // Verify that the creator cannot delete the commented asset
+                    AssetsTestUtil.assertDeleteAssetFails(client1, course, asset2.id, 401, function() {
+
+                      // Create an asset with likes
+                      AssetsTestUtil.assertCreateLink(client1, course, 'UC Riverside', 'http://www.ucr.edu/', null, function(asset3) {
+                        AssetsTestUtil.assertLike(client2, course, asset3.id, true, function() {
+                          // Verify that the creator cannot delete the liked asset
+                          AssetsTestUtil.assertDeleteAssetFails(client1, course, asset3.id, 401, function() {
+
+                            // Create an asset for use in a whiteboard
+                            AssetsTestUtil.assertCreateLink(client1, course, 'UC Riverside', 'http://www.ucr.edu/', null, function(asset4) {
+                              UsersTestUtil.assertGetMe(client1, course, null, function(user1Me) {
+                                WhiteboardsTestUtil.assertCreateWhiteboard(client1, course, 'UC Riverside Whiteboard', [user1Me.id], function(whiteboard) {
+                                  WhiteboardsTestUtil.addAssetToWhiteboard(client1, course, asset4, whiteboard, function() {
+                                    // Verify that the creator cannot delete the used asset
+                                    AssetsTestUtil.assertDeleteAssetFails(client1, course, asset4.id, 401, function() {
```

Comment:
Can we add a test case for an asset that's part of an exported whiteboard but no longer in use in an active whiteboard?